plugins/inputs/jolokia/jolokia.go

```diff
@@ -41,20 +44,32 @@ func (c JolokiaClientImpl) MakeRequest(req *http.Request) (*http.Response, error
 type Jolokia struct {
 	jClient JolokiaClient
 	Context string
+	Mode    string
 	Servers []Server
 	Metrics []Metric
+	Proxy   Server
 }
 
 func (j *Jolokia) SampleConfig() string {
 	return `
-  ## This is the context root used to compose the jolokia url
-  context = "/jolokia/read"
```

Comment:
'/jolokia' is actually the context. 'read' is the operation. I started to use POST request to support proxy mode and I moved the operation type into the request body.